tools/cpp/unix_cc_configure.bzl

```diff
@@ -528,6 +521,12 @@ def configure_unix_toolchain(repository_ctx, cpu_value, overriden_tools):
                 cpu_value,
                 False,
             )),
+            "%{cxx_standard}": escape_string(get_env_var(
+                repository_ctx,
+                "BAZEL_CXXOPTS",
```

Comment:
I don't think this is a safe change as previously this env var could have contained much more than just this. I think to keep compatibility you'll have to move this back up to where it was, just default it to nothing, and hardcode the C++ default here (although I guess it also could have affected the CXX include directories output? I'm not sure if that's good or bad)

Reply:
Ah, of course 🤦🏼 .

So since that's the case I've left everything in `unix_cc_configure.bzl` alone and only added the new features that set the standard.

If the features are activated then they take precedent over the `-std=c++0x` opt defined by `BAZEL_CXX_OPTS` (since they come later on the cmd line).

If one really doesn't want duplicate options then I guess they could set `BAZEL_CXXOPTS=""`.

Does that sound better? Certainly simpler implementation wise.

Reply:
yea sg